controller/tests/test_plan_controller.py

```diff
@@ -0,0 +1,147 @@
+import unittest
+from datetime import date
+
+from models.camp import Camp
+from models.plan import Plan
+from controller.plan_controller import close_plan
+from controller.plan_controller import find_plan
+from controller.plan_controller import create_plan
+from controller.plan_controller import list_plans
+from controller.plan_controller import create_camps
+from controller.plan_controller import controller_error
+
+
+class PlanControllerTest(unittest.TestCase):
+    def setUp(self):
+        Plan.delete_all()
+
+    def test_plan_status(self):
+        """
+        Test status flag of plan. Should be false is plan is open.
+        """
+        plan = Plan(name='My Plan',
+                    emergency_type=Plan.EmergencyType.EARTHQUAKE,
+                    description='Test emergency plan',
+                    geographical_area='',
+                    camps=[Camp(name='TestCamp')])
+        self.assertEqual(False, plan.close_plan_status)
+        self.setUp()
+
+    def test_close_plan_controller(self):
+        """
+        Test status flag of plan. Should be true when plan is closed.
+        """
+        plan = Plan(name='My Plan',
+                    emergency_type=Plan.EmergencyType.EARTHQUAKE,
+                    description='Test emergency plan',
+                    geographical_area='',
+                    camps=[Camp(name='TestCamp')])
+        close_plan(plan)
+        self.assertEqual(True, plan.close_plan_status)
+        self.setUp()
+
+    def test_no_close_date(self):
+        """
+        Test that close date is not set during plan creation.
+        """
+        plan = Plan(name='My Plan',
+                    emergency_type=Plan.EmergencyType.EARTHQUAKE,
+                    description='Test emergency plan',
+                    geographical_area='',
+                    camps=[Camp(name='TestCamp')])
+        self.assertEqual(None, plan.close_date)
+        self.setUp()
+
+    def test_close_date_set(self):
```

Comment:
It think it should be named `test_close_plan_controller`

Reply:
I already have a test_close_plan_controller test in this file but I can change it to be:
```
test_close_date_set_controller
```
Or combine the tests so that test_close_plan_controller checks that the flag is set to closed and the close_date is set as well.

Which would you prefer?

Reply:
I see, either will work then